var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -4,43 +4,56 @@
 class Julia(Package):
     """The Julia Language: A fresh approach to technical computing"""
     homepage = "http://julialang.org"
-    url      = "http://github.com/JuliaLang/julia/releases/download/v0.4.2/julia-0.4.2.tar.gz"
+    url      = "https://github.com/JuliaLang/julia/releases/download/v0.4.3/julia-0.4.3-full.tar.gz"
 
-    version('0.4.3', '7b9f096798fca4bef262a64674bc2b52')
-    version('0.4.2', 'ccfeb4f4090c8b31083f5e1ccb03eb06')
+    version('master',
+            git='https://github.com/JuliaLang/julia.git', branch='master')
+    version('0.4.5', '69141ff5aa6cee7c0ec8c85a34aa49a6')
+    version('0.4.3', '8a4a59fd335b05090dd1ebefbbe5aaac')
 
     patch('gc.patch')
+    patch('openblas.patch', when='@0.4:0.4.5')
 
-    # Build-time dependencies
-    depends_on("cmake @2.8:")
+    # Build-time dependencies:
     # depends_on("awk")
     # depends_on("m4")
     # depends_on("pkg-config")
-    depends_on("python @2.6:2.9")
 
-    # I think that Julia requires the dependencies above, but it builds find (on
-    # my system) without these. We should enable them as necessary.
+    # Combined build-time and run-time dependencies:
```

Comment:
Julia has a package manager. As long as Julia packages are installed by Julia (and not via Spack), many of these are actually run-time dependencies since the package manager needs them.

With "combined" I mean that they are both build-time and run-time dependencies.